src/main/java/graphql/nadel/Nadel.java

```diff
@@ -40,11 +42,22 @@
     private final Map<String, TypeDefinitionRegistry> typesByService = new LinkedHashMap<>();
     private final Map<String, SchemaNamespace> namespaceByService = new LinkedHashMap<>();
 
+    public Nadel(String dsl,
```

Comment:
we need javadoc here. Or maybe make constructor private and use Builder and document each of the fields there?